distdir_deps.bzl

```diff
@@ -246,22 +246,24 @@ DIST_DEPS = {
         ],
     },
     "com_google_absl": {
-        "archive": "20211102.0.tar.gz",
-        "sha256": "dcf71b9cba8dc0ca9940c4b316a0c796be8fab42b070bb6b7cab62b48f0e66c4",
+        "archive": "20220623.1.tar.gz",
```

Comment:
Do you mind also help submitting this version to https://github.com/bazelbuild/bazel-central-registry/tree/main/modules/abseil-cpp? So that we can also upgrade abseil to the same version with Bzlmod enabled.